### PR TITLE
Giulio/resolve conflicts merge 2015 11 10

### PR DESCRIPTION
**Pull request by caesar2164**
@dcadams & @stvstnfrd - resolutions of merge conflicts.

### CODE REVIEW

File: cms/templates/widgets/footer.html [hunk 1]
```diff
@@ -19,18 +19,6 @@
           <li class="nav-item nav-peripheral-pp">
             <a data-rel="edx.org" href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
           </li>
-<<<<<<< HEAD
-          % if settings.TENDER_DOMAIN and user.is_authenticated():
-          <li class="nav-item nav-peripheral-feedback">
-            <a data-rel="edx.org" href="http://${settings.TENDER_DOMAIN}/discussion/new" class="show-tender" title="${_('Use our feedback tool, Tender, to share your feedback')}">${_("Contact Us")}</a>
-          </li>
-          % elif user.is_authenticated():
```
Comment (stvstnfrd) [inline cms/templates/widgets/footer.html]: This shouldn't disappear; we added it for Zendesk.

Change `elif` to `if`.

Comment (caesar2164) [inline cms/templates/widgets/footer.html]: We can put the contact email in the sock by setting `PARTNER_SUPPORT_EMAIL` in features

File: cms/templates/widgets/sock.html [hunk 2]
```diff
@@ -69,28 +37,22 @@ <h3 class="title">${_("Request help with {studio_name}").format(studio_name=sett
...
-        }, {
-            'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
-            'sr_mouseover_text': _('Enroll in StudioX: Creating a Course with edX Studio'),
-            'text': _('Enroll in StudioX'),
-            'condition': True
-        }, {
             'href': 'mailto:{email}'.format(email=partner_email),
             'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
             'text': _('Contact Us'),
             'condition': 'PARTNER_SUPPORT_EMAIL' in settings.FEATURES
```
Comment (caesar2164) [inline cms/templates/widgets/sock.html]: We can add our support email here

File: cms/templates/settings.html [hunk 3]
```diff
@@ -307,7 +307,7 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                     %>${text}</%def>
                   <span class="tip tip-stacked">${overview_text()}</span>
                 </li>
-                
+
```
Comment (stvstnfrd) [inline cms/templates/settings.html]: Unrelated change; let's back this out;

File: cms/templates/widgets/footer.html (hunk 1 above)
Comment (stvstnfrd) [inline cms/templates/widgets/footer.html]: Ok, can you PR that? How about straight to configuration, instead of secure?

File: cms/templates/widgets/footer.html [hunk 4]
```diff
@@ -55,4 +43,3 @@
     </div>
   </footer>
 </div>
-
```
Comment (stvstnfrd) [inline cms/templates/widgets/footer.html]: Unrelated change; let's back this out;

File: cms/templates/widgets/sock.html [hunk 5]
```diff
@@ -69,28 +37,22 @@ <h3 class="title">${_("Request help with {studio_name}").format(studio_name=sett
...
+            'sr_mouseover_text': _('Access Course Staff Support on the Help Center to submit or review support tickets'),
+            'text': _('Help Center'),
+            'condition': True
+        }, {
             'href': 'https://open.edx.org',
             'sr_mouseover_text': _('Access the Open edX Portal'),
             'text': _('Open edX Portal'),
             'condition': not is_edx_domain
         }, {
-            'href': 'https://www.edx.org/course/overview-creating-edx-course-edx-edx101#.VO4eaLPF-n1',
```
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: We already had this link on our instance. It's a free, self-paced course about course creation.
Any particular reason to remove it?

File: cms/templates/widgets/sock.html [hunk 6]
```diff
@@ -69,28 +37,22 @@ <h3 class="title">${_("Request help with {studio_name}").format(studio_name=sett
...
-            'sr_mouseover_text': _('Enroll in edX101: Overview of Creating an edX Course'),
-            'text': _('Enroll in edX101'),
-            'condition': True
-        }, {
-            'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
```
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: Why don't we want this?

File: cms/templates/widgets/sock.html (hunk 5 above)
Comment (caesar2164) [inline cms/templates/widgets/sock.html]: This sends people to edX and they'll have to create an account there to be able to take the course.
We can keep them, but I was just removing clutter in the list of buttons/links.

File: cms/templates/widgets/sock.html (hunk 6 above)
Comment (caesar2164) [inline cms/templates/widgets/sock.html]: This sends people to edX and they'll have to create an account there to be able to take the course.
We can keep them, but I was just removing clutter in the list of buttons/links.

**Comment by stvstnfrd** [comment]
Review complete.

File: cms/templates/widgets/sock.html (hunk 5 above)
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: Sure, I think there's a discussion to be had as far as whether that's a feature or a bug.

Conflict resolution isn't the time to make that decision, though.

File: cms/templates/widgets/sock.html (hunk 6 above)
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: See above.

File: cms/templates/widgets/footer.html (hunk 1 above)
Comment (caesar2164) [inline cms/templates/widgets/footer.html]: https://github.com/Stanford-Online/configuration/pull/62

File: cms/templates/widgets/sock.html (hunk 5 above)
Comment (caesar2164) [inline cms/templates/widgets/sock.html]: cool, I'll remove both these changes

**Comment by stvstnfrd** [comment]
lgtm

**Comment by stvstnfrd** [comment]
@dcadams BTW: After we have resolved everything, let's be sure to fix the history/blame with https://github.com/Stanford-Online/edx-platform/wiki/protip-merge-smoothly
